src/libraries/System.Private.CoreLib/src/System/String.cs

```diff
@@ -432,14 +432,12 @@ public unsafe void CopyTo(int sourceIndex, char[] destination, int destinationIn
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void CopyTo(Span<char> destination)
         {
-            if ((uint)Length <= (uint)destination.Length)
```

Comment:
can you revert to the previous order? because you see diffs from that and it doesn't allow to detect regressions from non-elided bound checks. The order shouldn't matter since PGO is going to reorder it its way anyway, also, the current order looks more efficient to me (rarely taken branches are moved down)